### PR TITLE
Shona docu1102

### PR DESCRIPTION
**Pull request by shonaisbister-disco**

### CODE REVIEW

**Review by lucasdigital — APPROVED** [review]
Cleaner and clearer.

File: docs/support.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+id: support
+title: LiveData Migrator Support
+sidebar_label: Support
+---
+
+[WANdisco Support](https://community.wandisco.com/portal/s/) helps you manage all your interactions with WANdisco's support team, raise support cases, search our Knowledge Base, ask questions, and interact with our growing customer community.
```
Comment (stepgerner) [inline docs/support.md:7]: My brain might just be haywire, but I find "WANdisco Support helps you manage all your interactions with WANdisco's support team" to be weird. Maybe better saying "The WANdisco Support page"

File: docs/support.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Contacting Support
+
+The best way for you to reach our 24x7 Support team is to raise a case on the [WANdisco Support page](https://community.wandisco.com/portal/s/) by clicking on the **Get Support** link. However if your issue is urgent, call us using one of the following phone numbers.
```
Comment (stepgerner) [inline docs/support.md:11]: is "24x7" a thing? I think you can escape a forward slash by preceding it with a backslash, like 24\/7. Not sure though.

**Review by stepgerner — COMMENTED** [review]
Good stuff! Two comments.

**Pull request merged**